packages/cln_plugin/test/cln_plugin_method_test.dart

```diff
@@ -95,6 +113,48 @@ void main() {
       expect(getManifest.containsKey('dynamic'), true);
       expect(getManifest.containsKey('hooks'), true);
       expect(getManifest.containsKey('subscriptions'), true);
+
+      expect(getManifest["rpcmethods"], isNotNull);
+      expect(getManifest["options"], isNotNull);
+      expect(getManifest["notifications"], isNotNull);
+      expect(getManifest["dynamic"], isNotNull);
+      expect(getManifest["hooks"], isNotNull);
+      expect(getManifest["subscriptions"], isNotNull);
+    });
+  });
+  group('Init test: ', () {
+    test('Property test - dynamic', () {
+      var plugin = Plugin(
+          dynamic: true,
+          onInit: (plugin) {
+            return Future.value({});
+          });
+      var request = {
+        "options": {
+          "greeting": "World",
+          "number": [0]
+        },
+        "configuration": {
+          "lightning-dir": "/home/user/.lightning/testnet",
+          "rpc-file": "lightning-rpc",
+          "startup": true,
+          "network": "testnet",
+          "feature_set": {
+            "init": "02aaa2",
+            "node": "8000000002aaa2",
+            "channel": "",
+            "invoice": "028200"
+          },
+          "proxy": {
+            "type": "ipv4",
+            "address": "127.0.0.1",
+            "port": 9050
+          },
+          "torv3-enabled": true,
+          "always_use_proxy": false
+        }
+      };
+      plugin.init(plugin, request);
```

Comment:
when I try to access properties like `options` and `configuration`, it shows them as empty.

Reply:
The option inside the configuration is not the option that is registered inside the plugin.

You need to register the option `World` inside the plugin, or remove it from the configuration. 

We don't check this missing key in our code because core lightning send us the option that we specify in the manifest so this error should never happens.

Now the correct fix for this is the following one.

1. Add a test with the option registered in the plugin
2. Add a test without option inside the configuration
3. Make a sanity check that the key (optionName) exist inside the init method to avoid generic error like the one that you have in the test